app/forms/candidate_interface/reference/referee_name_form.rb

```diff
@@ -0,0 +1,19 @@
+module CandidateInterface
+  class Reference::RefereeNameForm
+    include ActiveModel::Model
+
+    attr_accessor :name
+
+    validates :name, presence: true
```

Comment:
This is a replacement for the validation that is `unless`'s in the main `application_reference` model, but it's less restrictive. It should also check for length, like the other one.

What if we kept the validation in the model, but used validation scopes? (mentioned below as well)